### PR TITLE
Prepare for docker

### PR DESCRIPTION
**Pull request by chalx**

### CODE REVIEW

**Comment by chalx** [comment]
@daltonfury42  please review

File: simplq/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 WORKDIR /build
 COPY . .
+
 RUN npm install
-RUN npm build
+RUN npm run build
 
 FROM nginx:1.19.6-alpine
 
 COPY nginx.conf /etc/nginx/nginx.conf
```
[the remark this answers is not in the document]
Comment (chalx) [reply simplq/Dockerfile:17]: I will check. Maybe 😂

[the remark this answers is not in the document]
Comment (chalx) [reply simplq/Dockerfile:17]: @daltonfury42 nginx.conf is  already commited. https://github.com/SimplQ/simplQ-frontend/blob/master/simplq/nginx.conf

[the remark this answers is not in the document]
Comment (daltonfury42) [reply simplq/Dockerfile:17]: Sorry, missed it. 😅

File: .github/workflows/docker_publish.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      uses: docker/setup-buildx-action@v1
+    
+    - name: Build and push
+      id: docker_build
+      uses: docker/build-push-action@v2
+      with:
+        context: ./simplq
+        file: .Dockerfile
+        push: true
+        tags: ${{ secrets.DOCKER_HUB_USERNAME }}/simpliQ-frontend:latest
```
Comment (daltonfury42) [inline .github/workflows/docker_publish.yml:31]: Created [daltonfury42/simplq-frontend ](https://hub.docker.com/repository/docker/daltonfury42/simplq-frontend), and added DOCKER_HUB_USERNAME and DOCKER_HUB_ACCESS_TOKEN to org secrets.

Can you rename to the repo name

**Comment by chalx** [comment]
@daltonfury42 done

**Review by daltonfury42 — APPROVED** [review]

**Pull request merged**